### ISSUE TITLE
Feature Request: Excluded URLs option

### ISSUE THREAD

**Issue by ElijahLynn** [igrigorik/videospeed#81]
I see the Videospeed over my hangouts video. 
![selection_713](https://cloud.githubusercontent.com/assets/1504756/12620025/94fff234-c4e8-11e5-9af5-a2b7aad5f318.png)

Here is the way Vimium does it:
![selection_714](https://cloud.githubusercontent.com/assets/1504756/12620053/b43b6d90-c4e8-11e5-86a3-e6aafa38c9b8.png)

**Comment by igrigorik** [comment]
Doh. We do exclude hangouts already, but looks like they changed the URLs? 

https://github.com/igrigorik/videospeed/blob/master/manifest.json#L27

I don't think user-defined exclude patterns is the right solution here. It doesn't make sense to enable this on hangouts period.

**Comment by ElijahLynn** [comment]
Ahh, the URL for the Hangouts is https://plus.google.com/hangouts which I think is the same. I see now in manifest.js that it is the same URL. After some sleuthing I realized I had sideloaded VS directly from a Git checkout and was on 0.2.4. But looks like the commit to add hangouts should have been in 0.2.4. 

I checked out master, reloaded and it seems to not show now. But then I switched back to 3848d4cb4cb0b8acd92cf521d487d4ee6d4fe1c0, reloaded again and it didn't show either.

So, not sure what was going on but glad to see it gone!
